build.gradle

```diff
@@ -104,7 +132,309 @@ subprojects { project ->
     tasks.withType(Test) {
         useJUnitPlatform()
     }
+
+    ext {
+        isExample = project.name.startsWith('example')
+        isGrailsPlugin = project.name.startsWith('grails-plugin')
+        pomInfo = {
+            delegate.name project.title
+            delegate.description subproject.projectDesc
```

Comment:
`subproject` should probably be `project`

Reply:
There were three subproject sections after removing commonBuild.  I had consolidated them, but went back and kept the first two separate to address.  

subprojects { subproject ->

subprojects { project ->

subproject {